klient/k8s/yaml/yaml_test.go

```diff
@@ -0,0 +1,140 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package yaml
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"testing"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/e2e-framework/pkg/env"
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+	"sigs.k8s.io/e2e-framework/pkg/envfuncs"
+	"sigs.k8s.io/e2e-framework/pkg/features"
+)
+
+var (
+	testenv env.Environment
+)
+
+func TestCreateResourceFromYAML(t *testing.T) {
+	var testYAML string = filepath.Join("testdata", "example-configmap-1.yaml")
+	createYAML := features.New("unstructured/unstructured").WithLabel("env", "dev").
```

Comment:
Admittedly, I have trouble picking names for these features.

```suggestion
	createYAML := features.New("???").WithLabel("env", "dev").
```

Reply:
#55 should help me in this regard